consts/errors.js

```diff
@@ -105,7 +105,7 @@ const errors = {
   },
   DOCUMENT_NOT_FOUND: (document) => ({
     code: 'DOCUMENT_NOT_FOUND',
```

Comment:
maybe add document name in code, something like code:`${document.toUpperCase()}_NOT_FOUND`

Reply:
i do not think that its doable, coz we throw one error code, just pass a several model names in the message if they didn't pass validation